int8 log_kill
int16 num_points
float64 avg_energy
int16 minutes
int16 seconds